aten/src/ATen/native/native_functions.yaml

```diff
@@ -6749,49 +6749,49 @@
     CPU: foreach_tensor_div_list_kernel_slow_
     CUDA: foreach_tensor_div_list_kernel_cuda_
 
-- func: _foreach_add.ScalarList(Tensor[] tensors, float[] scalars) -> Tensor[]
+- func: _foreach_add.ScalarList(Tensor[] tensors, Scalar[] scalars) -> Tensor[]
```

Comment:
I don't think it breaks backward compatibility. Could you confirm? What would happen if a scalar other than float is passed into an old model, in the new code, unlikely?

Reply:
no BC because those are private unused APIs.